### PR TITLE
Add CheckFrontendConfigExistence to check-gke-ingress

### PR DESCRIPTION
**Pull request by songrx1997**
/assign @swetharepakula

### CODE REVIEW

File: cmd/check-gke-ingress/app/ingress/rule.go [hunk 1]
```diff
@@ -81,6 +83,19 @@ func CheckHealthCheckTimeout(beConfig *beconfigv1.BackendConfig, svcName string)
 	return report.Passed, fmt.Sprintf("BackendConfig %s/%s in service %s/%s healthcheck configuration is valid", beConfig.Namespace, beConfig.Name, beConfig.Namespace, svcName)
 }
 
+// CheckFrontendConfigExistence checks wether a FrontendConfig exists.
```
Comment (swetharepakula) [inline cmd/check-gke-ingress/app/ingress/rule.go:86]: nit: typo whether

**Review by swetharepakula — COMMENTED** [review]
one nit, otherwise LGTM

**Review by swetharepakula — COMMENTED** [review]
one nit, otherwise LGTM

**Review by swetharepakula — COMMENTED** [review]
one nit, otherwise LGTM

File: cmd/check-gke-ingress/app/ingress/rule.go (hunk 1 above)
Comment (songrx1997) [reply cmd/check-gke-ingress/app/ingress/rule.go]: Updated, thanks!

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**